### PR TITLE
Change deploy.sh for Minikube example

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/250

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/252)

### CODE REVIEW

File: examples/MinikubeDemo/deploy.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ kubectl apply -f ../../manifests/modeldb/backend
 kubectl apply -f ../../manifests/modeldb/frontend
```
Comment (YujiOshima) [inline examples/MinikubeDemo/deploy.sh]: manifests/modeldb has been deleted. These lines should be deleted.

**Comment by YujiOshima** [comment]
@andreyvelich Thank you! I put a comment.

**Comment by andreyvelich** [comment]
@YujiOshima Done

**Comment by YujiOshima** [comment]
Thanks!! 
/lgtm
/approve

**Pull request merged**